assets/pages/bundles-landing/components/stackedBundle.jsx

```diff
@@ -236,6 +236,24 @@ function getDigitalAttrs(subsLinks: SubsUrls): SubscribeAttrs {
   return Object.assign({}, bundles.digital, { ctaLink: subsLinks.digital });
 }
 
+function WhyContribute(props: {shouldEncourageMonthly: boolean}) {
```

Comment:
Just a picky detail, there is a component called [WhySupport](https://github.com/guardian/support-frontend/blob/master/assets/components/whySupport/whySupport.jsx) so in order to avoid confusion, I would rename with another name.